src/middlewared/middlewared/plugins/disk.py

```diff
@@ -1243,7 +1246,7 @@ def check_clean(self, disk):
                     if name is None:
                         # Which means maximum has been reached and we can stop
                         break
-                    await run('gmirror', 'create', '-b', 'prefer', name, part_a, part_b)
+                    await run('gmirror', 'create', '-F', name, part_a, part_b)
```

Comment:
Why have you removed `-b prefer` from here?  IIRC it is needed to read dump from mirror.  Or middleware extracts it from raw disk(s)?

Reply:
-b was probably needed there before, when we labeled swaps statically.  If now we run savecore on individual disk, then it makes no sense now, same as -F.

Reply:
I included -F to set NOFAILSYNC on the mirror:
```
root@freenas[~]# gmirror list
Geom name: swap0
State: COMPLETE
Components: 2
Balance: load
Slice: 4096
Flags: NOFAILSYNC
GenID: 0
SyncID: 1
ID: 1942297127
Type: MANUAL
Providers:
...
```
I can remove it if that is not what we want.

Reply:
Well, whats the point in using NOFAILSYNC if the mirror wont exist anymore after reboot?